src/sections/Company/About/index.js

```diff
@@ -14,9 +14,37 @@ import CNCFstackedlogo from "./images/cncf-stacked-color.svg";
 import CNCFLightstackedlogo from "./images/cncf-light-stacked-color.svg";
 
 import { Link } from "gatsby";
+import { graphql, useStaticQuery } from "gatsby";
+import { getImage } from "gatsby-plugin-image";
+import { BgImage } from "gbimage-bridge";
+
 const Community_meetup = "./images/Community_meetup.png";
-const Layer5Projects = "./images/layer5-projects.png";
+// const Layer5Projects = "./images/layer5-projects.svg";
+
+
+
 const About = ({ theme }) => {
+
+  // Lee Calcote learned this trick from Maintainer Nikhil Ladha
+  const { webpImage } = useStaticQuery(
+    graphql`
+      query {
```

Comment:
You can use this query to fetch all the images after adding the path of about images in the gatsby-config.js file. 

`  {
      resolve: "gatsby-source-filesystem",
      options: {
        name: "about-images",
        path: `${__dirname}/src/sections/Company/About/images`,
      },
    },`



`{
  allFile(filter: { sourceInstanceName: { eq: "about-images" } }) {
    nodes {
      name
      childImageSharp {
        gatsbyImageData(width: 2000, quality: 50, webpOptions: { quality: 70 })
      }
    }
  }
}
`

Reply:
Naa, not need to use it that way when we are querying using the relative path key.
It can simply be a file name, where the path should be the relative path from the root directory of the sourced directories.
In this case simply `layer5-projects.png`